### DISCUSSION TITLE
New WebView bypass

### DISCUSSION THREAD

**Discussion by Blobby-Boi** [3kh0/ext-remover#972] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
So I found out a new WebView bypass for chrome web store apps. I’ve known about this for a while but my district blocked it so might as well make it public.

You need to have an app called “S2Bot 4 Scratch” which you can find [here](https://chrome.google.com/webstore/detail/s2bot-4-scratch/pllkalmkifgmanfoghenhgafbcpbicdj?hl=en-GB)
After you open it click on the dropdown menu on the top right and then pick the option that says “more information”. You should see a window pop up. Click on the logo that says “PICAXE” on the top left. Scroll down a bit to find the section called “Community and forum”. Click on the link below it. Scroll all the way down and on the right click on the email icon. It should open the google sign-in page. Type in “spstudent1@d11.org” and sign-in. Then click sign-in options and sign-in with GitHub. Click forgot password and then click on the GitHub logo on the top. Search google and then click on the second link.

This supports multiple windows.

**Comment by v4pl** [comment]
um i wonder if it works

**Reply by Blobby-Boi** [reply]
It should

**Reply by EliteNinja2009** [reply]
Recently, they've removed the location of the Google link but I've managed to work around it by searching "www.google.com", then, they removed the ability to click on the GitHub icon after "forgot password" but you just have to scroll down and click through a couple of links. Now, the entire method is patched as the people that control the district removed the option to sign in through GitHub rendering the entire thing useless. However, I have an idea that may work. We just need to find an email address of an organization that has GitHub as a sign in option and we should be able to access it again

**Reply by Blobby-Boi** [reply]
nah you can just search urlredirector in GitHub and use the repo I made a few years ago that still works

**Reply by EliteNinja2009** [reply]
Wow, I've also just did some moving around the links so I managed to get back to GitHub but it takes a while now. Thanks so much.  I'll be able to survive school this year!

**Comment by ghost** [comment]
ummmm my admin blocked it.

**Reply by S-PScripts** [reply]
Join our discord :) https://github.com/3kh0/ext-remover/discussions/1532

**Comment by schoolexploitkid** [comment]
it might also work with the 127.0.0.1:8871 or whatever method but thats not skid friendly
